README.md

```diff
@@ -121,6 +121,44 @@ MyComponent.propTypes = {
 export default withSnackbar(MyComponent);
 ```
 
+# Loading Mask
+
+There should be a unique loading mask for the whole app, so we need to insert a single provider in the main component:
+
+```
+import { LoadingProvider }  from "d2-ui-components";
+
+const MyAppWithLoadingMask = (
+    <LoadingProvider>
+        <MyApp />
+    </LoadingProvider>
+);
+```
+
+To use it, create a HOC with `withLoading`, add `loading` to your propTypes, and show the mask using the functions `props.loading.show()`.
+
+```
+import { withLoading } from "d2-ui-components";
+import PropTypes from "prop-types";
+
+const MyComponent = ({name, loading}) => (
+    <div>
+        <a onClick={() => loading.show()}>Show loading mask</a>
+        <a onClick={() => loading.show(false)}>Hide loading mask</a>
```

Comment:
loading.show(boolean) is useful, but can we have also `loading.hide()`?

Reply:
Sounds good, will add.